turbo.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This is the basic file from https://turbo.build/repo/docs/getting-started/create-new#3-understanding-turbojson

If you want a monorepo from scratch to harden the concepts I've described here, that page has a lot more details!

Reply:
Got it!

I wanted to get into Turborepo but I did not. Looks like I made a mistake by not going for it.

Thank you for the humbling code review and for fixing some of the issues. Make sure to take some time off from burning your eyes out with this radioactive spaghetti. 

Also, I found your https://www.shew.dev/monorepos and it looks like a great resource, definitely bookmarked, and will be diving into it shortly.

I need to step up my monorepo game. Thank you once again.